tools/setup-buildtools-mac.sh

```diff
@@ -24,8 +24,11 @@ sudo chown -R $(whoami) /usr/local/var/homebrew
 sudo chown -R $(whoami) /usr/local/etc/bash_completion.d /usr/local/include /usr/local/lib/pkgconfig /usr/local/share/aclocal /usr/local/share/locale /usr/local/share/zsh /usr/local/share/zsh/site-functions /usr/local/var/homebrew/locks
 
 brew install cmake
+brew install coreutils
 brew install wget
 brew install clang-format
 brew install google-benchmark
 brew tap nlohmann/json
 brew install nlohmann-json
+brew install abseil
```

Comment:
Technically we do not need `abseil` on Mac (only for old Mac, and/or only for old build with an old C++14 gcc on Mac). Recent Apple clang included has (almost complete) support for `std::variant` .